src/main/java/com/meetup/server/auth/presentation/AuthController.java

```diff
@@ -13,4 +19,14 @@
 @RequestMapping("/auth")
 public class AuthController {
 
+    private final AuthService authService;
+
+    @Operation(summary = "access token 재발급", description = "access token 재발급을 진행합니다.")
+    @GetMapping("/reissue")
+    public ApiResponse<?> reIssueToken(
```

Comment:
```suggestion
    public ApiResponse<ReissueTokenResponse> reIssueToken(
```
와 같이 응답이 있을 경우 타입을 고정하고, 없을 경우에 와일드카드(?)를 쓰는게 좋을 거 같아요!